cms/djangoapps/contentstore/views/tests/test_course_index.py

```diff
@@ -456,12 +450,13 @@ def test_separate_archived_courses(self, separate_archived_courses, username, or
         features = settings.FEATURES.copy()
         features['ENABLE_SEPARATE_ARCHIVED_COURSES'] = separate_archived_courses
         with override_settings(FEATURES=features):
-            self.check_index_page_with_query_count(separate_archived_courses=separate_archived_courses,
-                                                   org=org,
-                                                   mongo_queries=mongo_queries,
-                                                   sql_queries=sql_queries)
+            self.check_index_page_with_query_count(
+                separate_archived_courses=separate_archived_courses,
+                org=org,
+                mongo_queries=mongo_queries,
+                sql_queries=sql_queries,
+            )
```

Comment:
Was anything changed here? If not, can we drop it to avoid unnecessary changes in this commit?